src/main/java/com/yahoo/memory/Buffer.java

```diff
@@ -91,6 +91,32 @@ public static Buffer wrap(final ByteBuffer byteBuf, final ByteOrder byteOrder) {
    */
   public abstract Buffer region();
 
+  /**
+   * A region is a read-only view of the backing store of this object.
+   * This returns a new <i>Buffer</i> representing the defined region
+   * with the given offsetBytes, capacityBytes and byte order.
+   * <ul>
+   * <li>Returned object's origin = this objects' origin + <i>offsetBytes</i></li>
+   * <li>Returned object's <i>start</i> = 0</li>
+   * <li>Returned object's <i>position</i> = 0</li>
+   * <li>Returned object's <i>end</i> = <i>capacityBytes</i></li>
+   * <li>Returned object's <i>capacity</i> = <i>capacityBytes</i></li>
+   * <li>Returned object's <i>start</i>, <i>position</i> and <i>end</i> are mutable and
+   * independent of this object's <i>start</i>, <i>position</i> and <i>end</i></li>
+   * <li>Returned object's byte order = <i>byteOrder</i></li>
+   * </ul>
+   * If this object's capacity is zero, the returned object is effectively immutable and
+   * the backing storage and endianness are unspecified.
+   *
+   * <p><b>Note: </b><i>asMemory()</i> will return the originating <i>Memory</i> byte order.</p>
```

Comment:
Please rephrase it sounds like `asMemory()` returns a ByteOrder object.

Reply:
Done.